ui/src/app/machines/views/MachineDetails/MachineTests/MachineTestsTable/MachineTestsTable.tsx

```diff
@@ -118,7 +127,14 @@ const renderActions = (
     if (!hasVisibleMetrics) {
       links.push({
         children: "View metrics",
-        onClick: () => toggleMetrics(result.id),
+        onClick: () => {
+          toggleMetrics(result.id);
+          sendAnalytics(
+            "Machine testing",
+            "View testing script metrics",
```

Comment:
This should be in the same group as the action above as maybe just change this one too history instead of metrics